layouts/ucf-resource-link-classic.php

```diff
@@ -0,0 +1,303 @@
+				</div>
+			</div>
+
+		<?php
+		foreach ( $sections as $id => $section ):
+			$hide = false;
+			switch ( $id ) {
+				case 'resource-search-alpha':
+					if ( $args['default_sorting'] == 'term' ) {
+						$hide = True;
+					}
+					break;
+				case 'resource-search-term':
+					if ( $args['default_sorting'] == 'alpha' ) {
+						$hide = True;
+					}
+					break;
+			}
+	?>
+			<div class="<?php echo $id; ?>"<?php if ( $hide ) { echo ' style="display:none;"'; } ?>>
+				<div class="row">
+				<?php
+				$count = 0;
+				foreach ( $section as $section_title => $section_posts ):
+					if ( count( $section_posts ) > 0 || $args['show_empty_sections'] ):
+				?>
+
+					<?php if ( $section_title == $args['uncategorized_term_name'] ): ?>
+						</div>
+							<div class="row">
+								<div class="<?php echo $args['column_width']; ?>">
+									<div class="resource-search-heading-wrap" id="jump-to-<?php echo strtolower( esc_html( $section_title ) ); ?>">
+										<h3 class="resource-search-heading"><?php echo esc_html( $section_title ); ?></h3>
+										<hr>
+									</div>
+								</div>
+							</div>
+
+							<div class="row">
+							<?php
+							// $split_size must be at least 1
+							$split_size = max( floor( count( $section_posts ) / $args['column_count'] ), 1 );
+							$split_posts = array_chunk( $section_posts, $split_size );
+							foreach ( $split_posts as $resource => $column_posts ):
+							?>
+								<div class="<?php echo $args['column_width']; ?>">
+									<ul class="resource-search-list">
+									<?php foreach( $column_posts as $key => $post ): ?>
+										<li data-post-id="<?php echo $post->ID; ?>">
+											<?php echo UCF_Resource_Link_PostType::toHTML( $post ); ?><?php echo $section_title; ?>
+										</li>
+									<?php endforeach; ?>
+									</ul>
+								</div>
+							<?php endforeach; ?>
+
+					<?php else: ?>
+
+						<?php if ( $count % $args['column_count'] == 0 && $count !== 0 ): ?>
+							</div><div class="row">
+						<?php endif; ?>
+
+						<div class="<?php echo $args['column_width']; ?>">
+							<div class="resource-search-heading-wrap" id="jump-to-<?php echo strtolower( esc_html( $section_title ) ); ?>">
+								<h3 class="resource-search-heading"><?php echo esc_html( $section_title ); ?></h3>
+								<div class="back-to-top"><a href="#top">Back to Top</a></div></div>
+							<hr>
+							<ul class="resource-search-list">
+							<?php foreach( $section_posts as $post ): ?>
+								<li data-post-id="<?php echo $post->ID; ?>">
+									<?php echo UCF_Resource_Link_PostType::toHTML( $post ); ?><span class="hidden-xs-up"><?php echo $section_title; ?></span>
+								</li>
+							<?php endforeach; ?>
+							</ul>
+						</div>
+
+				<?php
+						endif;
+					$count++;
+					endif;
+				endforeach;
+				?>
+				</div><!-- .row -->
+			</div><!-- term/alpha section -->
+
+		<?php endforeach; ?>
+
+		</div><!-- .resource-search -->
+	<?php
+		$content = ob_get_clean();
```

Comment:
It looks like in this function `$content` gets completely wiped out by the output of the function? What exactly is `$content` supposed to be?